login.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="main-container">
+      <form class="login-container" action="books-list.html">
```

Comment:
No need for the `action` 😄

Reply:
I did it just to make it more "real" just for fun 😄